Library/Formula/ansiweather.rb

```diff
@@ -0,0 +1,18 @@
+class Ansiweather < Formula
+  homepage "https://github.com/fcambus/ansiweather"
+  url "https://github.com/fcambus/ansiweather/archive/1.02.tar.gz"
+  sha256 "6b65ba86a0a3af7ac0fbfea3c0e871371c519d0fafd79e4a517909128bf7ca5f"
+
+  head "https://github.com/fcambus/ansiweather.git"
+
+  depends_on "jq"
+
+  def install
+    bin.install "ansiweather"
+  end
+
+  test do
+    output = shell_output("#{bin}/ansiweather -l Ithaca,NY")
+    assert_match /Current weather in Ithaca/, output
```

Comment:
Doubtful, I can't find any references to environment variables in `ansiweather`'s documentation or source code.